spec/receptor/api_object_spec.rb

```diff
@@ -85,6 +85,32 @@
         expect(response.body).to eq(body)
       end
     end
+
+    context "async" do
+      # Not implemented yet
+    end
+  end
+
+  describe "#post" do
+    let(:post_data) { {:service_parameters => {"username" => "user1", "quest" => "Test Topology", "airspeed" => 50}, :provider_control_parameters => {}} }
+
+    context "sync" do
+      it "sends request to the default path and type" do
+        body = {:key => :value}
+        expect(subject).to(receive(:send_request)
+                             .with(:post, '/api/v2/job_templates', :data => post_data)
```

Comment:
@slemrmartin shouldn't this include the job_template_id?

Reply:
This is testing the common `post` method, so it doesn't matter in this spec, but I'll change it.
The ordering spec exists, anyway I'll try to extend it

Reply:
hm, this method was intended to be generic but it seems it doesn't make much sense, that's why the launch() method is using `send_request` directly. I'll remove it